Core/clim-core/gadgets.lisp

```diff
@@ -2624,10 +2639,9 @@ if INVOKE-CALLBACK is given."))
           ;;       which I could have easily worked around without adding kludges
           ;;       to the scroller-pane..
           (let* ((topmost-pane (if scroll-p
-                                  ;list-pane
                                   (scrolling (:scroll-bar :vertical
-                                              :suggest-height height   ;; Doesn't appear to be working..
-                                              :suggest-width (if scroll-p (+ 30 (bounding-rectangle-width list-pane))))
+                                              :suggested-height height
```

Comment:
Please explain this change in the commit message or even make a separate commit.

Reply:
The change is referred to in the commit message: fixed typos in popup-init (`:suggest-` -> :`suggested-`). The effect is so minor I didn't think it deserved a separate commit.

Reply:
I only saw that in the description of the pull request, not the commit message itself. It is generally useful to include all relevant information in the commit message.

Reply:
please amend commit description, there is no need for a separate commit, but noting it in the commit message makes sense